doc/tools/make_rst.py

```diff
@@ -376,6 +399,21 @@ def main():  # type: () -> None
     )
     args = parser.parse_args()
 
+    # Retrieve heading translations for the given language.
+    if not args.dry_run:
+        lang_file = os.path.join(
+            os.path.dirname(os.path.realpath(__file__)), "..", "translations", "{}.po".format(args.lang)
+        )
+        if os.path.exists(lang_file):
+            import polib
```

Comment:
Not a requirement, but we can use something like this to produce a nicer error message when `polib` is missing.

```python
try:
    import polib
except ImportError:
    print("Section header localization requires `polib`.")
    exit(1)
```